### PR TITLE
Exercise 10 of chapter 11 completed.

### PR DESCRIPTION
**Pull request by Htocki**

### CODE REVIEW

File: bin/11_10/src/Main.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  buffer << string;
+  std::vector<std::string> substrings;
+  std::copy(
+    std::istream_iterator<std::string> { buffer },
+    std::istream_iterator<std::string> { },
+    std::back_inserter(substrings));
+  return substrings;
+}
+
+void Print(const std::vector<std::string> vector) {
```
Comment (sialiuk) [inline bin/11_10/src/Main.cpp]: should be reference

File: bin/11_10/src/Main.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  std::copy(
+    std::begin(vector),
+    std::end(vector),
+    std::ostream_iterator<std::string> { std::cout, "\n" });
+}
+
+int main() {
+  const std::string string {
```
Comment (sialiuk) [inline bin/11_10/src/Main.cpp]: why not constexpr std::string_view

**Review by sialiuk — CHANGES_REQUESTED** [review]

File: bin/11_10/src/Main.cpp (hunk 1 above)
Comment (Htocki) [reply bin/11_10/src/Main.cpp]: Fixed.

File: bin/11_10/src/Main.cpp (hunk 2 above)
Comment (Htocki) [reply bin/11_10/src/Main.cpp]: Fixed.

**Review by sialiuk — APPROVED** [review]

**Pull request merged**